docs/user-guide/command-line-interface.md

```diff
@@ -30,62 +30,62 @@ The command line utility has several options. You can view the options by runnin
 eslint [options] file.js [file.js] [dir]
 
 Basic configuration:
-  --no-eslintrc                  Disable use of configuration from .eslintrc.*
-  -c, --config path::String      Use this configuration, overriding .eslintrc.* config options if present
-  --env [String]                 Specify environments
-  --ext [String]                 Specify JavaScript file extensions - default: .js
```

Comment:
Unrelated change seems to have snuck in?

Reply:
Interesting. What I did here was, I ran `./bin/eslint.js --help` and replaced this entire section with the output, which is the same method I used for previous PRs I submitted of this type. I should have reviewed the diff myself and left a comment explaining this. However, as of right now, `eslint --help` does not include "default: .js" here.

Let me know if you prefer I open a new PR with only a replacement of this help section with the current `eslint --help` output, without the other changes I made intentionally.

Reply:
Fair enough. Just wanted to double check.